### PR TITLE
Typo in comment

### PR DESCRIPTION
**Pull request by ggreif**

### CODE REVIEW

File: interpreter/valid/valid.ml [hunk 1]
```diff
@@ -49,7 +49,7 @@ let label (c : context) x = lookup "label" c.labels x
  * Note: The declarative typing rules are non-deterministic, that is, they
  * have the liberty to locally "guess" the right types implied by the context.
  * In the algorithmic formulation required here, stack types are hence modelled
- * as lists of _options_ of types here, where `None` representss a locally
+ * as lists of _options_ of types here, where `None` represents a locally
```
Comment (ggreif) [inline interpreter/valid/valid.ml]:
```suggestion
 * as lists of _options_ of types, where `None` represents a locally
```
"here" already appears in the previous line.

Comment (rossberg) [reply interpreter/valid/valid.ml]: Agreed.

**Comment by ggreif** [comment]
This can be squashed and merged.

**Comment by rossberg** [comment]
Thanks!

**Pull request merged**